<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="m_clientIp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.2.128</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Addr:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_eepromAddr">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x1ff000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Len:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_eepromLen">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnReadEeprom">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QHexEdit" name="m_eepromDump" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>621</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnWriteEeprom">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnEraseEeprom">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnEditNetworkSettings">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>20</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Network Settings</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>./Dump/qhexedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
